ts/WoltLabSuite/Core/Ui/Poll/Manager/View/Participants.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Abstract implementation for participants views.
+ *
+ * @author  Joshua Ruesweg
+ * @copyright  2001-2022 WoltLab GmbH
+ * @license  GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @module  WoltLabSuite/Core/Ui/Poll/Manager/View/Participants
+ * @since   5.5
+ */
+
+import Manager from "../Manager";
+import UiUserList from "../../../User/List";
+
+export class Participants {
+  protected readonly pollManager: Manager;
+  private button: HTMLButtonElement;
+  private userList?: UiUserList;
```

Comment:
```suggestion
  private userList?: UiUserList = undefined;
```

Defining properties on runtime is somewhat error-prone and has the drawback of nuking the prototype inside the JS engine. Please feel free to compare the difference it makes in the JS output to understand what is happening with this change.